### ISSUE TITLE
vdiff under a custom path not working anymore

### ISSUE THREAD

**Issue by elakito** [tjsnell/vdiff#3]
Hi Tracy,
it looks like the resource references and path rewriting in the current version assumes the app to be deployed at the root. So the instruction at README is not working, as it uses the path parameter to deploy the app at path version-diff. 

> You can run within your web container or alternatively run it standalone using jetty-runner
> 
> java -jar target/dependency/jetty-runner.jar --path version-diff --port 8181 target/*.war
> 
> Then, using your browser to open
> 
> http://localhost:8181/version-diff/

It looks like the change (or this limitation) came during the previous marge.

Will you be looking into this? It's not urgent, though.

regards, aki

**Comment by tjsnell** [comment]
Yeah. I ran into issues not having it run at root that I need to explore. The README needs updating. It’s on my list. SimpleServlet that adds in the URL support is what pushed that root requirement and I just haven’t had time to sit and think about it :)

Last night I ran into githubs 60 api calls/hr limit. Didn’t realize it existed.
